drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -60,9 +60,26 @@ static struct nrf5_802154_data nrf5_data;
 #define NRF5_802154_CFG(dev) \
 	((const struct nrf5_802154_config * const)(dev)->config_info)
 
+#if CONFIG_IEEE802154_VENDOR_OUI
+#define IEEE802154_NRF5_VENDOR_OUI CONFIG_IEEE802154_VENDOR_OUI
+#else
+#define IEEE802154_NRF5_VENDOR_OUI (uint32_t)0xF4CE36
+#endif
```

Comment:
Not a block

You can define at Kconfig.nrf the default value, since its 15.4 global scope
config IEEE802154_VENDOR_OUI
    default 0xF4CE36

When you want to change it, add below at app.conf file, for instance.
CONFIG_IEEE802154_VENDOR_OUI=XXXX

Reply:
I believe that it's ok to keep this value in the driver's sources since it's a constant. With the solution proposed above (IEEE802154_VENDOR_OUI is a boolean) we can easily determine if this option is used or not.